### ISSUE TITLE
text-decoration: underline 0.1em; shouldn't be an error

### ISSUE THREAD

**Issue by 1jj** [validator/validator#1165]
URL being validated or code to reproduce error:
http://jeka.info/test/Finger/

```
    Error: CSS:  text-decoration: 0.1em  is not a color value.
    From line 11, column 43; to line 11, column 48
    underline 0.07em; text
```
AFAICT `text-decoration: underline 0.1em;` is valid and works in Firefox and Chromium.

**Comment by nschonni** [comment]
https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration

> Adds text-decoration-thickness; note that this isn't yet officially part of the shorthand — this is as yet unspecified.

**Comment by 1jj** [comment]
https://drafts.csswg.org/css-text-decor-4/#text-decoration-property
This has consensus in the CSSWG and has three independent implementations (Chromium/Blink, Firefox/Gecko, Safari/Webkit).

This warning isn't helpful for anyone.

**Comment by sideshowbarker** [comment]
Thanks for raising this. I’ve added a workaround to the HTML checker but the real fix needs to be made in the CSS-checking backend — so it would be great if you could also raise an issue at https://github.com/w3c/css-validator/issues/new

**Comment by 1jj** [comment]
>  — so it would be great if you could also raise an issue at https://github.com/w3c/css-validator/issues/new

Does that make sense? They don't have a "CSS level 4" profile ...

**Comment by sideshowbarker** [comment]
> Does that make sense? They don't have a "CSS level 4" profile ...

For the CSS checker, the “CSS3” profile in practice just means “current version” — there’s no plan to ever add a separate level 4
